microproxy/layer/application/http1.py

```diff
@@ -93,44 +138,87 @@ def on_response(self, response):
             request=self.req, response=response)
 
         self.resp = plugin_result.response if plugin_result else response
+
+    def send_response(self):
         try:
-            self.src_conn.send_response(self.resp)
+            if int(self.resp.code) in range(200, 600):
```

Comment:
I think there is no need to change range into xrange with following reason:
- range between 200 to 600 is not a big amount
- xrange had been removed in python3

So I think it's not too much of benefit that we could take from change it to xrange!
What do you think?